packages/core/src/components/icon/icon.tsx

```diff
@@ -26,7 +26,7 @@ export class Icon {
   @Prop({ reflect: true }) name: IconNames = 'truck';
 
   /** Pass a size of icon as a string, for example, 32px, 1rem, 4em... */
-  @Prop({ reflect: true }) size: string = '16px';
+  @Prop({ reflect: true }) size: string = undefined;
```

Comment:
I know its still in progress, but I think you can just do :

```suggestion
  @Prop({ reflect: true }) size?: string;
```
or 

```suggestion
  @Prop({ reflect: true }) size: string;
```